src/test/java/rife/database/TestDbQueryManager.java

```diff
@@ -30,16 +29,12 @@ public void tearDown(Datasource datasource) {
             var connection = datasource.getConnection();
 
             // drop the test table
-            var statement = connection.createStatement();
-            try {
+            try (var statement = connection.createStatement()) {
                 try {
                     statement.executeUpdate(new DropTable(datasource).table("tbltest"));
                 } catch (DatabaseException e) { /* don't do anything */ }
-            } finally {
-                try {
-                    statement.close();
-                } catch (DatabaseException e) { /* don't do anything */ }
```

Comment:
This was intended to make tests not fail in case the test table has an error on tear down, changing it with a try resource and then letting the `DatabaseException` throw will not do that. This change also leaves a meaningless dangling comment line below.